musicpalette/musicpalette/SwipeSelectorViewController.swift

```diff
@@ -58,9 +58,10 @@ class SwipeSelectorViewController: UIViewController, SwipeSelectorDelegate {
             return
         } else {
             // TODO: Add Playlist View Controller at the overflow item
-            let playlistViewController = UIViewController()
-            playlistViewController.view.backgroundColor = UIColor.green
-            self.present(playlistViewController, animated: true, completion: nil)
+            let pc:PlaylistTableViewController = PlaylistTableViewController(swipeModels: rightSwipes)
+            pc.view.backgroundColor = UIColor.green
+            PlaylistTableViewController.
+            self.present(PlaylistTableViewController, animated: true, completion: nil)
```

Comment:
You will want to present the instance of `PlaylistTableViewController`, but it looks like you're trying to present the class. You've created the instance `pc` above.

You should be able to replace with
```
self.present(pc, animated: true, completion: nil)
```